package_nso_to_oc/xe/xe_ospfv2.py

```diff
@@ -555,6 +569,68 @@ def set_timers(intf, intf_leftover):
     
     return {"openconfig-network-instance:config": config}
 
+def set_authentication(intf, intf_leftover):
+    # Authentication type: unconfigured, null, simple and md5
+    authentication = {"openconfig-ospfv2-ext:config": []}
+    auth_list = authentication["openconfig-ospfv2-ext:config"]
+    is_auth_enabled = "ip" in intf and "ospf" in intf["ip"] and "authentication" in intf["ip"]["ospf"]
```

Comment:
In the NED, SIMPLE is an empty dict. is_auth_enabled won't be True if the authentication type is SIMPLE.